tests/scripts/build_minimal_appimage_with_custom_apprun.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+if [[ -z ${ARCH+x} ]]; then
+    echo "ARCH not set."
+    exit -1;
+fi
+
+LINUXDEPLOY=$1
+if [[ -z ${LINUXDEPLOY+x} ]]; then
+    echo "LINUXDEPLOY path not set."
+    exit -1;
+fi
+
+RUNNABLE=$2
+if [[ -z ${RUNNABLE+x} ]]; then
+    echo "RUNNABLE path not set."
+    exit -1;
+fi
+
+DESKTOP_FILE=$3
+if [[ -z ${DESKTOP_FILE+x} ]]; then
+    echo "DESKTOP_FILE path not set."
+    exit -1;
+fi
+
+ICON=$4
+if [[ -z ${ICON+x} ]]; then
+    echo "ICON path not set."
+    exit -1;
+fi
+
+
+LINUXDEPLOYDIR=`mktemp -d`
+pushd "$LINUXDEPLOYDIR"
+
+    cp "$LINUXDEPLOY" .
+
+    wget https://github.com/TheAssassin/linuxdeploy-plugin-appimage/releases/download/continuous/linuxdeploy-plugin-appimage-"$ARCH".AppImage
```

Comment:
Such hackish stuff that adds dependencies to external projects is not acceptable within the scope of unit tests. We're not attempting to perform integration tests. And we certainly don't want to add a dependency to other fragile projects.

We're not trying to show off here...

Reply:
Dropped